### PR TITLE
Initial fix to ES-Reader to help prevent timeouts.

### PR DESCRIPTION
**Pull request by smashew**
fixes the reader problems with timeouts.

### CODE REVIEW

File: streams-contrib/streams-persist-elasticsearch/src/main/java/org/apache/streams/elasticsearch/ElasticsearchPersistReader.java [hunk 1]
```diff
@@ -63,6 +62,8 @@ public ElasticsearchPersistReader(ElasticsearchReaderConfiguration config) {
     public ElasticsearchPersistReader(ElasticsearchReaderConfiguration config, ElasticsearchClientManager escm) {
         this.config = config;
         this.elasticsearchClientManager = escm;
+        int batchSize = config.getBatchSize().intValue() <= 0 ? 10 : config.getBatchSize().intValue();
```
Comment (robdouglas) [inline streams-contrib/streams-persist-elasticsearch/src/main/java/org/apache/streams/elasticsearch/ElasticsearchPersistReader.java]: Can we move the default batch size up to a const?

File: streams-contrib/streams-persist-elasticsearch/src/main/java/org/apache/streams/elasticsearch/ElasticsearchQuery.java [hunk 2]
```diff
@@ -99,6 +94,10 @@ public void setBatchSize(int batchSize) {
     }
 
     public void setScrollTimeout(String scrollTimeout) {
+        this.scrollTimeout = TimeValue.parseTimeValue(scrollTimeout, TimeValue.timeValueMillis(5 * 1000 * 60));
```
Comment (robdouglas) [inline streams-contrib/streams-persist-elasticsearch/src/main/java/org/apache/streams/elasticsearch/ElasticsearchQuery.java]: Is there a default value being used for the scrollTimeout in the event that the user does not specify one?

**Comment by rbnks** [comment]
:+1:

File: streams-contrib/streams-persist-elasticsearch/src/main/java/org/apache/streams/elasticsearch/ElasticsearchQuery.java (hunk 2 above)
Comment (rbnks) [inline streams-contrib/streams-persist-elasticsearch/src/main/java/org/apache/streams/elasticsearch/ElasticsearchQuery.java]: In the jsonschema a default value is set.

Comment (smashew) [inline streams-contrib/streams-persist-elasticsearch/src/main/java/org/apache/streams/elasticsearch/ElasticsearchQuery.java]: Yes, there is a default in the schema. Default in that call too.. The second parameter is if the default fails.

**Comment by smashew** [comment]
:+1:

**Comment by robdouglas** [comment]
:+1:

**Pull request merged**